notebook/READEME.md

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
name of this file should be README.md (typo)

Reply:
Corrected the typo.